### PR TITLE
Bump version to 0.5.0

### PR DESCRIPTION
**Pull request by rnestler**
## Checklist

- [x] Tests added if applicable
- [x] README updated if applicable
- [x] CHANGELOG updated if applicable
- [x] If a commit includes a breaking change, include the string `[breaking-change]` somewhere in the commit message

### CODE REVIEW

**Comment by rnestler** [comment]
Hmm the changelog is actually not completely up to date :/

**Comment by rnestler** [comment]
> Hmm the changelog is actually not completely up to date :/

Updated it, should be fine now :slightly_smiling_face:

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +24,7 @@ Possible log types:
 - [added] Add usage and API documentation (#94)
 - [changed] Some internal types that were never meant to be
   used outside the crate are now not public anymore (#94)
+- [added] Allow to set custom redis pool for custom options (#90)
```
Comment (dbrgn) [inline CHANGELOG.md]: Do we want to mention #86 as well?

> - [changed] Drop dependency on `rustc_serialize` (#86)

**Comment by dbrgn** [comment]
Can you wait until #104 is merged?

**Comment by dbrgn** [comment]
#104 is merged, so this PR can be rebased :)

**Comment by dbrgn** [comment]
@rnestler ping! would be great if this could be shipped soon, since the docs on docs.rs are quite incomplete.

File: CHANGELOG.md (hunk 1 above)
Comment (rnestler) [reply CHANGELOG.md:29]: Fixed.

**Comment by rnestler** [comment]
> @rnestler ping! would be great if this could be shipped soon, since the docs on docs.rs are quite incomplete.

Updated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ Possible log types:
 
 ### Unreleased
 
+### v0.5.0 (2019-12-30)
```
Comment (dbrgn) [inline CHANGELOG.md:16]: Date needs updating.

Comment (rnestler) [reply CHANGELOG.md]: Hehe, we only needed slightly more than 3 month to get the release done :laughing:

**Pull request merged**